src/main/java/toDo.java

```diff
@@ -0,0 +1,15 @@
+public class toDo extends task{
+    protected String description;
+
+    public toDo(String description) throws DukeException{
+        this.description=description;
+        if(this.description.isEmpty()==true||this.description.trim().isEmpty()==true){
```

Comment:
This [compound expression](https://nus-cs2113-ay2122s1.github.io/website/se-book-adapted/chapters/codeQuality.html#avoid-complicated-expressions) might be hard to read. Perhaps do the boolean expression in multiple steps?